contracts/hyphen/LiquidityPool.sol

```diff
@@ -87,6 +87,7 @@ contract LiquidityPool is
     event TrustedForwarderChanged(address indexed forwarderAddress);
     event LiquidityProvidersChanged(address indexed liquidityProvidersAddress);
     event TokenManagerChanged(address indexed tokenManagerAddress);
+    event BaseGasUpdated(uint256 indexed baseGas);
```

Comment:
Verified that events order doesn't affect the storage so while upgrading events can be added in any order
https://forum.openzeppelin.com/t/using-events-in-upgradeable-contracts/2381